README.md

```diff
@@ -68,9 +68,19 @@ release_directory: /Users/Example_User/Dropbox/release/
 
 Yes. We have custom builders for Python and R. You can also use them with the same syntax as the Stata builder. If you're using R, make sure it's been added to your PATH and that you have a YAML-parsing package, such as [yaml](https://cran.r-project.org/web/packages/yaml/yaml.pdf). 
 
+##### Can I pass "command line style" arguments to a script?
+
+You bet. All of our custom builders accept "command line style" arguments with the same method. Enumerate the arguments in a list and pass them to the builder through the `CL_ARG` keyword argument, exactly the same way you specify sources and targets. We'll format this list, and `scons` will pass its contents to the script at runtime. You can reference these arguments when writing a script using the standard practice for its language.
```

Comment:
Didn't we get rid of the example to make the template easier to use from scratch.